test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,40 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void ifAnyVoid() {
```

Comment:
Good that you attempted last week's LO. However, avoid accumulating unrelated changes into a single PR. 

Also, are **void** and **null** are the same in Java? If they aren't the same, do you think the method name should be changed to better reflect the test?

Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.